sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsModelFactory.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <summary> Model factory for read-only models. </summary>
+    [CodeGenType("AzureDigitalTwinsAPIModelFactory")]
+    public static partial class DigitalTwinsModelFactory
```

Comment:
I noticed that we had a hand-crafted "DigitalTwinsModelFactory" before. A recent check-in added the auto-generated model factory, which ends up being called "AzureDigitalTwinsAPIModelFactory". This customization updates the name to that of the previous model factory. Since the methods being generated are identical (plus one additional one), I've removed the hand-crafted one in favor of the auto-generated one.